content/klc/S2.3.adoc

```diff
@@ -1,21 +1,18 @@
 +++
-brief = "Where atomic parts are available in multiple footprint options, a separate symbol must be drawn for each footprint"
+brief = "Where parts are available in multiple footprint options, a separate symbol must be drawn for each footprint"
 +++
 
-Many electronic components are provided in multiple footprint (package) options. These _may or may not_ be pin compatible.
+Many electronic components are provided in multiple packaging variants. These _may or may not_ be pin compatible.
```

Comment:
I might change 'packaging variants' to 'packages' for brevity and clarity. At least it's more clear to me.

Reply:
done

Reply:
'packaging variant' is also used on line 7 and could go to 'package'.

Reply:
done